### DISCUSSION TITLE
Can 2fa be set up from a mobile device?

### DISCUSSION THREAD

**Discussion by Iceman248** [DefGuard/defguard#681] [Q&A] [answered]
I'm reading about the desktop application, but I can't tell for sure whether a mobile device can be set up to use 2fa to connect to a wire guard VPN.

**Comment by teon** [comment, marked as answer]
Hu @Iceman248,

We are planing to develop our own Mobile application - but have non now.
We're waiting for Tauri 2.0 which is our base framework for multi-platfrom desktop apps to finish beta and be production ready for mobile apps (which is comming in 2.0).

Thus MFA is not supported on mobile.
